README.md

```diff
@@ -1 +1,38 @@
-Используйте линтеры в своих домашних заданиях. Успехов!
\ No newline at end of file
+# This is http_server with long awaited (for me) cats and animals.
+## Firstly, you should execute commands:
+```bash
+    git clone https://github.com/siriusdevs/rpm-hws_7-2_2023
+
+    git checkout puskareva
+
+    cd homework_http_fork
+```
+
+## Secondly you need in Docker container and database
+
+```bash
+docker run -d \
+--name animals_db \
+-p :32771 \
+-e POSTGRES_USER=lina \
+-e POSTGRES_PASSWORD=cat \
+-e POSTGRES_DB=animals \
+postgres
+```
+
+```bash
+psql -h 127.0.0.1 -p 32771 -U lina -d animals_db
```

Comment:
this command only connects me to postgresql. here you should provide -f some_file.ddl, where some_file.ddl contains your initial sql commands to set up a db